app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  validates :name, presence: true
```

Comment:
- [ ]  [optional] user_name length should be 3 or more, then it will be better validation.

Reply:
done, thanks